app/main/openapi.yaml

```
@@ -17,16 +17,16 @@ info:
     bruke denne Swagger-dokumentasjonen som en referanse til å lage en automatisert test,
     ikke teste selve oppslaget.
     
-    Vi tilbyr to løsninger, avhengig av behov og konsument. Vi har kalt dem henholdsvis
-    minimumsløsningen og maksimumsløsningen.
+    Vi tilbyr tre løsninger, avhengig av behov og konsument. Vi har kalt dem henholdsvis
+    minimumsløsningen, medium- og maksimumsløsningen.
     
     For en gitt person returnerer maksimumsløsningen en liste med vedtak, gyldig for en
     gitt periode, og en liste med utbetalinger på dette vedtaket.
     
     
     | Ord                   | Kommentar                                                                                                                      |
     |-----------------------|--------------------------------------------------------------------------------------------------------------------------------|
-    | Dagsats               | Her har vi to verdier. Den som ligger på selve vedtaket er før samordning uten barnetillegg, mens det som ligger i utbetalingsobjektet er etter samordning uten barnetillegg. |
+    | Dagsats               | Her har vi to verdier. Den som ligger på selve vedtaket er før samordning uten barnetillegg, mens det som ligger i utbetalingsobjektet er etter samordning uten barnetillegg.  Om kildesystemet er Kelvin (nytt system), er dette full dagsats, uten reduksjoner. |
```

```suggestion
    | Dagsats               | Her har vi to verdier. Den som ligger på selve vedtaket er før samordning uten barnetillegg, mens det som ligger i utbetalingsobjektet er etter samordning uten barnetillegg. |
```

trenger ikke denne da det er samme case for begge kildesystemer.